src/ssis_migrator/parsers/component_parser.py

```
@@ -0,0 +1,248 @@
+            '{149447B8-8A7C-4FC7-B4E6-5DD2687916C1}': 'Data Conversion',
+            '{1E7B0B8A-8A7C-4FC7-B4E6-5DD2687916C1}': 'Lookup',
+            '{2E7B0B8A-8A7C-4FC7-B4E6-5DD2687916C1}': 'Merge Join',
+            '{3E7B0B8A-8A7C-4FC7-B4E6-5DD2687916C1}': 'Union All',
+            '{4E7B0B8A-8A7C-4FC7-B4E6-5DD2687916C1}': 'Sort',
+            '{5E7B0B8A-8A7C-4FC7-B4E6-5DD2687916C1}': 'Aggregate',
+            '{6E7B0B8A-8A7C-4FC7-B4E6-5DD2687916C1}': 'Conditional Split',
+            '{7E7B0B8A-8A7C-4FC7-B4E6-5DD2687916C1}': 'Multicast',
+            '{8E7B0B8A-8A7C-4FC7-B4E6-5DD2687916C1}': 'Script Component'
+        }
+    
+    def parse_data_flow_component(self, comp_elem: ET.Element, namespaces: Dict[str, str]) -> Optional[Dict[str, Any]]:
```

Pylint (3.3.7)

[refactor] Too many branches (21/12)

(R0912)